packages/packages/locations/src/__tests__/locations.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { render } from '@testing-library/react';
+import { lazy } from 'react';
+import { addFill, resetFills } from '../locations';
+import Slot from '../components/slot';
+import { expect } from '@playwright/test';
+
+describe( '@elementor/locations locations', () => {
+	beforeEach( () => {
+		resetFills();
+	} );
+
+	it( 'should render components based on the location name', () => {
+		addFill( {
+			location: 'test',
+			component: () => <div data-testid="element">First div</div>,
+		} );
+
+		addFill( {
+			location: 'test',
+			component: () => <div data-testid="element">Second div</div>,
+		} );
+
+		addFill( {
+			location: 'test2',
+			component: () => <div data-testid="element">Should not exists</div>,
+		} );
+
+		const { getAllByTestId } = render( <Slot location="test" /> );
+
+		const elements = getAllByTestId( 'element' );
+
+		expect( elements ).toHaveLength( 2 );
+		expect( elements[ 0 ].innerHTML ).toBe( 'First div' );
+		expect( elements[ 1 ].innerHTML ).toBe( 'Second div' );
+	} );
+
+	it( 'should render components based on priority', () => {
+		addFill( {
+			location: 'test',
+			component: () => <div data-testid="element">Third div</div>,
+		} );
```

Comment:
maybe add a comment that the default is 10?

Reply:
Done